src/utils/operator.ts

```diff
@@ -12,14 +12,17 @@ import { formattingAmountPrecision, formatSmartMaxPrecision } from 'utils'
 import { PENDING_ORDERS_BUFFER } from 'apps/explorer/const'
 
 function isOrderFilled(order: RawOrder): boolean {
+  const { kind, executedBuyAmount, sellAmount, executedSellAmount, buyAmount, executedFeeAmount, executedSurplusFee } =
+    order
   let amount, executedAmount
 
-  if (order.kind === 'buy') {
-    amount = new BigNumber(order.buyAmount)
-    executedAmount = new BigNumber(order.executedBuyAmount)
+  const totalFee = new BigNumber(executedFeeAmount).plus(new BigNumber(executedSurplusFee))
```

Comment:
Not for this PR, but what about returning the `total` fee from the SDK directly and hide this logic?

Reply:
yes, we have a lot of duplication 😓

Reply:
@anxolin If you wish I can help with it

Reply:
https://github.com/cowprotocol/cow-sdk/pull/88 - anyway, I took 2 mins to add to the SDK :)